src/ape_console/_cli.py

```diff
@@ -28,11 +28,12 @@
     context_settings=dict(ignore_unknown_options=True),
 )
 @network_option()
+@click.option("--embed", help="Use an embedded IPython", is_flag=True)
 @ape_cli_context()
-def cli(cli_ctx, network):
+def cli(cli_ctx, network, embed):
     """Opens a console for the local project."""
     verbose = cli_ctx.logger.level == logging.DEBUG
-    return console(verbose=verbose)
+    return console(verbose=verbose, embed=embed)
```

Comment:
Should we just always embed?

Reply:
then we can't do cool error handling things or any other fun IPython integration stuff.
we will need an exception handler to properly filter out irrelevant stack frames when showing source code tracebacks